src/main/java/core/gui/comp/CheckBoxTree/CheckBoxTree.java

```diff
@@ -0,0 +1,236 @@
+package core.gui.comp.CheckBoxTree;
+
+import javax.swing.*;
+import javax.swing.event.*;
+import javax.swing.tree.*;
+import java.awt.*;
+import java.awt.event.*;
+import java.util.EventListener;
+import java.util.EventObject;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+
+public class CheckBoxTree extends JTree {
```

Comment:
Hey @wsbrenk can you please give attribution to https://stackoverflow.com/a/21851201/289466 ?  SO code is [CC BY-SA](https://creativecommons.org/licenses/by-sa/4.0/)

Reply:
@tychobrailleur is that sufficient?

Reply:
Yes, thanks a mil!